chainer/optimizer.py

```diff
@@ -27,153 +30,316 @@ def exponential_decay_noise(xp, shape, dtype, hook, opt):
     return xp.random.normal(0, std, shape).astype(dtype)
 
 
-class Optimizer(object):
-    """Base class of all numerical optimizers.
+class Hyperparameter(object):
```

Comment:
Neither do I have a strong opinion. It's OK if you have an intent to choose this one. But I agree that it's better to ask native speakers.